app/models/user.rb

```diff
@@ -0,0 +1,25 @@
+class User < ApplicationRecord
+  before_save{self.email = email.downcase}
+
+  validates :name, presence: true,
+                    length: {
+                      maximum: Settings.validation.name.max_length
+                    }
```

Comment:
```suggestion
                    length: {maximum: Settings.validation.name.max_length}
```
Viết như này đi em trông cho gọn